tgui/packages/tgui/interfaces/IceCreamMachine.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author EleSeu (https://github.com/EleSeu)
+ * @license ISC
+ */
+
+import { useBackend } from '../backend';
+import { Button, Flex, Icon, Section, Stack } from '../components';
+import { Window } from '../layouts';
+import { ReagentGraph } from './common/ReagentInfo';
+
+export const StandardFlavors = (props, context) => {
+  const { act, data } = useBackend(context);
+  const standardFlavors = data.flavors;
+
+  return (
+    <Section title="Standard Flavors" >
+      <Flex wrap>
+        {standardFlavors.map((flavor, flavorIndex) => (
+          <Button
+            key={flavorIndex}
+            align="left"
+            width="130px"
+            m=".1rem"
+            onClick={() => act("make_ice_cream", { flavor: flavor.name })}
```

Comment:
All other dispensers like this have the reagent names capitalised

Reply:
i don't think this was addressed?

Reply:
Using className="chem-dispenser__dispense-buttons" makes the buttons capitalised